common/app/views/fragments/metaData.scala.html

```diff
@@ -1,4 +1,4 @@
-@(item: model.MetaData)(implicit request: RequestHeader)
+@(item: model.MetaData, amp: Boolean)(implicit request: RequestHeader)
```

Comment:
you can actually set the default to false to save having to pass it in
`@(item: model.MetaData, amp: Boolean = false)(implicit request: RequestHeader)`